scipy-1.0/paper.tex

```diff
@@ -249,6 +249,35 @@ \subsection*{Numerical optimization}
 
 
 \subsection*{Statistical distributions}
+Version 1.0 of SciPy saw the addition of the energy distance algorithm in
+\texttt{scipy.stats.energy\_distance}. Energy distance is used in a variety
+of applications ranging from genetic analysis\cite{Hu2008DetectingIC} to 
+multivariate independence\cite{szekely2009}. We also added the
+first Wasserstein distance algorithm to the library in
+\texttt{scipy.stats.wasserstein\_distance}. The Wasserstein distance is often
+used in color digital image analysis \cite{192468}.
+
+We now provide the cumulative distribution function of the multivariate
+normal distribution, \\\texttt{scipy.stats.multivariate\_normal}, via the newly
+associated \texttt{logcdf} and \texttt{cdf} methods. Other recent distributions 
```

Comment:
Maybe mention that cdf uses under the hood the Fortran code by Alan Genz (scipy/stats/mvndst.f) which probably everybody else is using.